docs/src/pages/components/text/react.mdx

```diff
@@ -97,19 +97,10 @@ import '@aws-amplify/ui-react/styles.css';
 
 <Example>
   <Text className="primary-styled-variable-text" variation="primary">
-    This is my styled text
+    {'This is my styled text'}
```

Comment:
This fixes an console error `p tag cannot be nested inside p tag`.

Reply:
Interesting..